### ISSUE TITLE
Progress bar glitch

### ISSUE THREAD

**Issue by hyperf0cus** [ultralytics/ultralytics#22679]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Ultralytics YOLO Component

Train

### Bug

<img width="885" height="289" alt="Image" src="https://github.com/user-attachments/assets/e02ad1d5-6193-4d9b-8f33-238003b451e3" />

Why progress bar while training looks like this by printing to each line? It worked well before, how can I fix?

### Environment

```Ultralytics 8.3.228 🚀 Python-3.11.11 torch-2.7.0+cu126 CUDA:0 (NVIDIA GeForce RTX 3060, 11940MiB)
Setup complete ✅ (8 CPUs, 62.6 GB RAM, 444.8/467.9 GB disk)

OS                     Linux-5.15.0-139-generic-x86_64-with-glibc2.31
Environment            Linux
Python                 3.11.11
Install                pip
Path                   /home/almatshakenov/environments/yolov12/lib/python3.11/site-packages/ultralytics
RAM                    62.64 GB
Disk                   444.8/467.9 GB
CPU                    Intel Core(TM) i7-9700 3.00GHz
CPU count              8
GPU                    NVIDIA GeForce RTX 3060, 11940MiB
GPU count              1
CUDA                   12.6

numpy                  ✅ 1.26.4>=1.23.0
matplotlib             ✅ 3.10.0>=3.3.0
opencv-python          ✅ 4.9.0.80>=4.6.0
pillow                 ✅ 10.4.0>=7.1.2
pyyaml                 ✅ 6.0.1>=5.3.1
requests               ✅ 2.32.3>=2.23.0
scipy                  ✅ 1.13.0>=1.4.1
torch                  ✅ 2.7.0>=1.8.0
torch                  ✅ 2.7.0!=2.4.0,>=1.8.0; sys_platform == "win32"
torchvision            ✅ 0.22.0>=0.9.0
psutil                 ✅ 5.9.8
polars                 ✅ 1.35.2
ultralytics-thop       ✅ 2.0.18>=2.0.18
```

### Minimal Reproducible Example

yolo detect train data=coco8.yaml model=yolo11n.yaml epochs=100 imgsz=640

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @hyperf0cus, thanks for reporting this and for using Ultralytics 🚀 This is an automated response to help you get support quickly—an Ultralytics engineer will follow up soon.

We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

Since you’re seeing a progress bar rendering issue, please also share:
-Exact command used (you shared: `yolo detect train data=coco8.yaml model=yolo11n.yaml epochs=100 imgsz=640` ✅)
-Terminal/shell and environment where this occurs (e.g., bash, zsh, tmux, VS Code terminal, Jupyter/Colab, systemd service)
-Whether the issue reproduces in a fresh terminal without multiplexers (tmux/screen) or logging
-Full console output of the first few epochs
-Any custom logging settings or environment variables
-A short screen recording or raw text log snippet showing the line-by-line printing behavior

If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [Python>=3.8](https://www.python.org/) environment with [PyTorch>=1.8](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

-**Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
-**Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
-**Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
-**Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

Thanks again for the detailed environment info and the screenshot 🙏 Once you share the additional details above, we’ll help narrow this down quickly.

**Comment by Y-T-G** [comment]
It's because your window is too small. You need to either decrease font size, or increase column size of your terminal
